pkg/storaged/content-views.jsx

```diff
@@ -526,25 +531,28 @@ function block_content(client, block, allow_partitions) {
     var format_disk_btn = null;
     if (allow_partitions)
         format_disk_btn = (
-            <div className="pull-right">
+            <div className="pull-right" key="create-partition-table">
                 <StorageButton onClick={format_disk} excuse={block.ReadOnly ? _("Device is read-only") : null}>
                     {_("Create partition table")}
                 </StorageButton>
             </div>);
 
     return (
         <CockpitListing.Listing title={_("Content")}
-                                actions={format_disk_btn}>
+                                actions={[ format_disk_btn ]}
+                                emptyCaption={_("No entries")}>
```

Comment:
The `emptyCaption` property isn't needed here since the content display will never be empty.  It either has a row for the full block device, or at least a row for the free space when there is a partition table.

Do we need to specify a value because `cockpit-components-listing.jsx` declares it as required?  Then we should just use a non-translated empty string here.  It doesn't make sense to ask people to translate a string that describes a impossible situation.

Reply:
Right, it's just because of that.
ok, makes sense to use empty string here